src/CabanaPD_Solver.hpp

```diff
@@ -327,13 +330,14 @@ class SolverElastic
     bool print;
 };
 
-template <class DeviceType, class ParticleType, class ForceModel,
-          class BoundaryCondition, class PrenotchType>
+template <class DeviceType, class InputType, class ParticleType,
+          class ForceModel, class BoundaryCondition, class PrenotchType>
 class SolverFracture
-    : public SolverElastic<DeviceType, ParticleType, ForceModel>
+    : public SolverElastic<DeviceType, InputType, ParticleType, ForceModel>
```

Comment:
@streeve : What is InputType?

Reply:
Same as above, `Inputs`, but now templated for the variable dimension